controller/projects.go

```diff
@@ -64,6 +65,8 @@ func (p *Projects) Create(user *model.User, input model.NewProject) (*model.Proj
 			strconv.Itoa(MaxProjectsLimit) + "projects reached.")
 	}
 
+	creationTime := time.Now()
```

Comment:
I feel this should be handled by database no?

Reply:
Didn't realize that was a thing! Changed.